### DISCUSSION TITLE
v17.9.1 Released! 🎉

### DISCUSSION THREAD

**Discussion by pH-7** [pH7Software/pH7-Social-Dating-CMS#1102] [Release Announcements]
Wooohooo, **pH7 Social Dating Builder, v17.9.1** has now been released!! 🪄

*pH7Builder v17.9.1 brings minor enhancements, bug fixes, and patches some issues occurring on the latest PHP versions, v8.1 / v8.2*

Enjoy 🤗 Hope you will love it! ⚡️

<hr /><em>This discussion was created from the release <a href='https://github.com/pH7Software/pH7-Social-Dating-CMS/releases/tag/v17.9.1'>v17.9.1 Released! 🎉</a>.</em>

**Comment by htrott** [comment]


**Reply by SosthenesAlpha** [reply]
How did u set it up to work on webserver sir ??

It works perfectly well on my localhost but does not work properly on webserver. 
The issue on the webserver is that users can't register properly. The option to select gender and other info (during registration) is not given. The script simply chooses gender as woman and takes users to login page 🤷🏿‍♂️🤷🏿‍♂️🤷🏿‍♂️. On localhost, it doesn't do this at all. It loads propely and runs well.

Many thanks for the awaited response.

**Comment by TheAlgoslingers** [comment]
Ahaha, it looked well in the screenshots but haphazard in reality
